client/views/login.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+<html id="html" lang="en">
+
+<head>
+    <meta charset="utf-8">
+</head>
+
+<body>
+    <form name="forms" id="forms" method="POST">
+        <div>
+            <label>Login
+                <input name="username" type="text" />
+            </label>
+        </div>
+        <div>
+            <label>Password
+                <input name="password" type="text" />
+            </label>
+        </div>
+        <button type="submit">Submit</button>
+    </form>
+    <span id='error' style="color: red"></span>
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+    <script>
+        const formElement = document.querySelector('form#forms')
+    
+        const getFormJSON = (form) => {

```

Comment:
If you use plain vanilla JS and fetch this would be far more simpler. JQuery is no longer needed in the modern browser/js world.

Do not depend on libraries when there is an standar ready available solution that's easy to use.